### PR TITLE
feat(file_explorer): remove '.' and rename '..' to '< Back'

### PR DESCRIPTION
**Pull request by AndreCostaaa**
This change tackles #7160 and https://github.com/lvgl/lvgl/issues/7160#issuecomment-2443950986 by removing the current directory entry '.' and replacing '..' with '< Back'

## Before:
image

## After:

image

### CODE REVIEW

**Comment by lhdjply** [comment]
I think removing "." is fine. However, changing ".." to "< back" might not be ideal because some people may use languages other than English. What do you think about changing it to "< .."?

**Comment by lhdjply** [comment]
I feel that only "<" seems a bit odd. Let's look at other opinions.

**Comment by kisvegabor** [comment]
> back" might not be ideal because some people may use languages other than English. What do you think about changing it to "< .."?

Note that DEVICES and PLACES are also English. I'm ok with `< Back` for now.

**Comment by lhdjply** [comment]
https://github.com/lvgl/lvgl/blob/f7fbcbd71493ecf9b6f60fdd64221b392e9175ef/src/others/file_explorer/lv_file_explorer.c#L26-L29  @AndreCostaaa  This section also needs to be modified.In addition, please merge the latest master commit and add the test of file_explorer.

> Note that DEVICES and PLACES are also English. I'm ok with < Back for now.

@liamHowatt Do you have any comment on that?

File: src/others/file_explorer/lv_file_explorer.c [hunk 1]
```diff
@@ -560,8 +557,7 @@ static void show_dir(lv_obj_t * obj, const char * path)
         return;
     }
 
-    lv_table_set_cell_value_fmt(explorer->file_table, index++, 0, LV_SYMBOL_DIRECTORY "  %s", ".");
-    lv_table_set_cell_value_fmt(explorer->file_table, index++, 0, LV_SYMBOL_DIRECTORY "  %s", "..");
+    lv_table_set_cell_value_fmt(explorer->file_table, index++, 0, LV_SYMBOL_LEFT "  %s", "Back");
```
Comment (liamHowatt) [inline src/others/file_explorer/lv_file_explorer.c]:
```suggestion
    lv_table_set_cell_value(explorer->file_table, index++, 0, LV_SYMBOL_LEFT "  " LV_FILE_NAVIGATION_PARENT_DIR);
```

**Review by liamHowatt — COMMENTED** [review]
In my opinion, "< Back" is a huge improvement. It's clear from the symbol that it's a special entry and "back" is I think a fine word to use since as kisvegabor said there are other english words used in the file explorer.

**Review by liamHowatt — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**